src/driver/odoc_unit.ml

```diff
@@ -195,6 +194,7 @@ let of_packages ~output_dir ~linked_dir ~index_dir (pkgs : Packages.t list) :
     let name = mld_path |> Fpath.rem_ext |> Fpath.basename |> ( ^ ) "page-" in
     let unit =
       make_unit ~name ~kind ~rel_dir ~input_file:mld_path ~pkg ~include_dirs
+        ~lib_deps:[]
```

Comment:
Is there anything bigger but still sensible we could use here?

- We likely cannot add the union of the the `lib_deps` from the libraries of the package: I think there might be different deps with the same name? (not sure here).

I know we will provide a way to extend this, but having a bigger default would be nice!

Reply:
I think the libraries in the package is what we decided before.